app/helpers/elmo_form_builder.rb

```diff
@@ -191,7 +191,9 @@ def elmo_field(field_name, options)
         @template.content_tag(:div, human_val, class: "ro-val", 'data-val': val != human_val ? val : nil)
 
       else
-        tag_options = options.slice(:id, :data) # Include these attribs with all tags, if given.
+        # TODO: Ideally, any additional options would pass through to all items
+        #   automatically without need to be allowed here explicitly.
+        tag_options = options.slice(:id, :data, :checked) # Include these attribs with all tags, if given.
```

Comment:
I allowed `:checked` to pass thru to the checkbox so now it works as you intended in ERB, and we don't need to check this box with JS (which was breaking one of the specs you had removed). This was a head-banger!